[2024-09-10T17:30:15.220+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T17:30:15.261+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers manual__2024-09-10T17:28:02.881148+00:00 [queued]>
[2024-09-10T17:30:15.280+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers manual__2024-09-10T17:28:02.881148+00:00 [queued]>
[2024-09-10T17:30:15.281+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 4
[2024-09-10T17:30:15.310+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2024-09-10 17:28:02.881148+00:00
[2024-09-10T17:30:15.326+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=62) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-10T17:30:15.332+0000] {standard_task_runner.py:72} INFO - Started process 63 to run task
[2024-09-10T17:30:15.331+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'max_min_dag', 'generate_random_numbers', 'manual__2024-09-10T17:28:02.881148+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/tmp/tmpzscly63p']
[2024-09-10T17:30:15.333+0000] {standard_task_runner.py:105} INFO - Job 109: Subtask generate_random_numbers
[2024-09-10T17:30:15.442+0000] {task_command.py:467} INFO - Running <TaskInstance: max_min_dag.generate_random_numbers manual__2024-09-10T17:28:02.881148+00:00 [running]> on host 0a0201be4888
[2024-09-10T17:30:15.603+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Denis' AIRFLOW_CTX_DAG_ID='max_min_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T17:28:02.881148+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-10T17:28:02.881148+00:00'
[2024-09-10T17:30:15.605+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T17:30:15.681+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-10T17:30:15.703+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T17:30:15.704+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=max_min_dag, task_id=generate_random_numbers, run_id=manual__2024-09-10T17:28:02.881148+00:00, execution_date=20240910T172802, start_date=20240910T173015, end_date=20240910T173015
[2024-09-10T17:30:15.753+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T17:30:15.880+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-10T17:30:15.885+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
